schemas/common_definitions.json

```diff
@@ -118,6 +118,14 @@
           "type": "string",
           "description": "A repeated group id, for which the count will be fetched."
         },
+        "type": {
+          "type": "string",
+          "description": "Type of rule used for when condition."
```

Comment:
This should be an enum containing "answer_count"

Reply:
Done